.github/workflows/test_prometheus_rules.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test prometheus rules
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  promtool:
+    runs-on: ubuntu-latest
```

Comment:
Is it possible to move to self-hosted CI?

Reply:
Not really, just more runners. But as you removed from the pull request path, as @zmraul suggested, I am fine keeping it here.